vector/src/main/java/im/vector/app/core/extensions/Activity.kt

```diff
@@ -114,6 +116,19 @@ fun AppCompatActivity.hideKeyboard() {
     currentFocus?.hideKeyboard()
 }
 
+fun AppCompatActivity.validateBackPressed(onBackPressed: () -> Unit) {
```

Comment:
Please add documentation to explain that this should be only used for top-level activities. A developer which is not aware of this could think that it's a good practice to use it for all activities.

Reply:
Done

Reply:
Funny: I had the same mental path "A developer which is not aware of this could think that it's a good practice to use it for all activities". And now I see this comment in GitHub, and the comment in the code :)